src/ert/config/ert_config.py

```diff
@@ -808,10 +810,9 @@ class ErtConfigWithPlugins(ErtConfig):
             PREINSTALLED_FORWARD_MODEL_STEPS: ClassVar[
                 Mapping[str, ForwardModelStep]
             ] = preinstalled_fm_steps
-            PREINSTALLED_WORKFLOWS = (
+            PREINSTALLED_WORKFLOWS = dict(
```

Comment:
There are still some changes here that are unrelated to moving workflows into the config module